nginx/default-local.conf

```diff
@@ -17,7 +17,7 @@ server {
     return 200 'gateway works';
   }
 
-  location / {
+  location /api/ {
```

Comment:
Is `/docs` or `/api/docs` still reachable with this (swagger docs)?

Reply:
`/docs` works for me

Reply:
Does not work for me. I'll check.

Reply:
This was reverted in favour of: https://github.com/citysciencelab/masterportal-scenario-explorer/pull/2